autoload/asyncomplete.vim

```diff
@@ -421,6 +426,10 @@ function! s:recompute_pum(...) abort
 
         if l:startcol > l:ctx['col']
             call asyncomplete#log('core', 's:recompute_pum', 'ignoring due to wrong start col', l:startcol, l:ctx['col'])
+            echomsg string(["'invalid cols'", {
```

Comment:
any reason why `echomsg` instead of just `asyncomplete#log`?

Reply:
Sorry... It is wrong commit! I'll revert it.